e2e/pod_priority_class_test.go

```diff
@@ -209,4 +299,162 @@ var _ = Describe("enforcing a Priority Class", func() {
 		}
 	})
 
+	// Ensure default field does not cause false positives for the previous tests
+	It("add default to tenant", func() {
+		tnt.Spec.PriorityClasses.Default = "tenant-default"
+	})
```

Comment:
This test assertion is useless, since there's no update at all: please, fix it with an update in the `It` block or drop it.